druid/src/env.rs

```diff
@@ -461,7 +461,14 @@ impl Data for EnvImpl {
 
 impl Default for Env {
     fn default() -> Self {
-        let l10n = L10nManager::new(vec!["builtin.ftl".into()], "./resources/i18n/");
+        Env::with_l10n(vec![])
+    }
+}
+
+impl Env {
+    pub(crate) fn with_l10n(mut resources: Vec<String>) -> Self {
+        resources.insert(0, "builtin.ftl".into());
+        let l10n = L10nManager::new(resources, "./resources/i18n/");
```

Comment:
so here is where our existing limitations are exposed: basically this only works if all these files are in the same place, and that's not what we should expect. In the longer term we want to figure this out, and allow for both built-in and custom resources, but I think what we should do for now is just allow the user to *replace* the built-in resources with their own. This isn't perfect but at least lets the user do what they need to do; they can copy over the builtin resources as needed.

Reply:
Fixed.